src/main/java/org/broadinstitute/dsm/jobs/PubSubLookUp.java

```diff
@@ -91,7 +91,9 @@ private static boolean shouldWriteResultIntoDB(TestBostonResult testBostonResult
 
     public static void tellPepperAboutTheNewResults(TestBostonResult testBostonResult) {
         logger.info("Going to Notify Pepper");
```

Comment:
How about a bit more context in the log, like `"Going to notify pepper about " + testBostonResults.getX...`

Reply:
That would contain PHI I think...